### PR TITLE
python tests

### PR DESCRIPTION
**Pull request by jburel**
Fix some integration tests

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-push#225](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-push/225/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-push/225/consoleText) for more details.
Possible conflicts:
  - PR #6108 will-moore 'Don't use wrap(id) for long IDs'
    - components/tools/OmeroPy/test/integration/test_scripts.py

~~--conflicts~~ Conflict resolved in build [OMERO-push#62](https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-push/62/). See the [console output](https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-push/62/consoleText) for more details.

**Comment by joshmoore** [comment]
@jburel : leaving you to fix the conflict and propose next steps.

**Comment by joshmoore** [comment]
Restarted travis due to github issues:
```
Error opening connection java.io.IOException: Server returned HTTP response code: 503 for URL: https://pypi.org/packages/source/o/omero-py/omero-py-5.5.dev2.tar.gz
```

File: components/tools/OmeroPy/test/integration/test_repository.py [hunk 1]
```diff
@@ -573,8 +574,8 @@ def testFakeImport(self):
         try:
             gateway._waitOnCmd(handle, failonerror=True)
             rsp = handle.getResponse()
-            assert dict == type(rsp.globalMetadata)
-            assert dict == type(rsp.seriesMetadata)
+            assert isinstance(type(rsp.globalMetadata), dict)
```
Comment (sbesson) [inline components/tools/OmeroPy/test/integration/test_repository.py]:
```suggestion
            assert isinstance(rsp.globalMetadata, dict)
```

File: components/tools/OmeroPy/test/integration/test_repository.py [hunk 2]
```diff
@@ -573,8 +574,8 @@ def testFakeImport(self):
...
+            assert isinstance(type(rsp.seriesMetadata), dict)
```
Comment (sbesson) [inline components/tools/OmeroPy/test/integration/test_repository.py]:
```suggestion
            assert isinstance(rsp.seriesMetadata, dict)
```

**Comment by joshmoore** [comment]
Getting there!

**Pull request merged**